app/serializers/event_serializer.rb

```diff
@@ -30,6 +29,13 @@ def estimated_distance
     Geo.distance_between(user_location, object)
   end
 
+  def exception_note
+    exception_notes = EventZipCode.exception_note(@instance_options[:zip_code],
```

Comment:
I'm concerned about using class methods that have queries in them within the serializer. I worry that this will make performance optimization more difficult in the future. Also, I don't really like the use of @instance_options in the serializer. It creates a tight coupling between the serializer and the consumers of the serializer. Future developers will have to be aware that they need to provide the zip_code option every time this serializer is invoked.